browser/main/SideNav/StorageItem.js

```diff
@@ -131,8 +132,64 @@ class StorageItem extends React.Component {
     }
   }
 
+  handleDragEnter (e) {
+    e.dataTransfer.setData('defaultColor', e.target.style.backgroundColor)
+    e.target.style.backgroundColor = 'rgba(129, 130, 131, 0.08)'
+  }
+
+  handleDragLeave (e) {
+    e.target.style.opacity = '1'
+    e.target.style.backgroundColor = e.dataTransfer.getData('defaultColor')
```

Comment:
Thanks for commenting @asmsuechan ! 
Even now, the background color of the folder that is hover while dragging is supposed to change. How do you want the background color to change?